src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/settings/SettingsPluginComponent.kt

```diff
@@ -0,0 +1,50 @@
+package nl.tudelft.ewi.se.ciselab.testgenie.settings
+
+import com.intellij.util.ui.FormBuilder
+import javax.swing.JCheckBox
+import javax.swing.JComponent
+import javax.swing.JPanel
+
+/**
+ * This class displays and captures changes to the values of the Settings entries.
+ */
+class SettingsPluginComponent {
+    var panel: JPanel? = null
+
+    //Checkbox options
+    private var showCoverageCheckBox = JCheckBox("Do you want visualised coverage? ")
```

Comment:
Perhaps this should be renamed to "Show visualised coverage" or similar instead of "Do you want visualised coverage?", as checkbox labels phrased as questions look weird and would not be consistent with the rest of the IntelliJ UI

Reply:
Thanks for the suggestion. Done.